concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -1,197 +1,362 @@
+        'Authorization'            : authorized_golem_transfer_token,
+        'Concent-Client-Public-Key': b64encode(CONCENT_PUBLIC_KEY).decode(),
+        'Concent-upload-path'      : '{}/{}/result'.format(task_id, part_id),
+        'Content-Type'             : 'application/x-www-form-urlencoded'
     }
+    return (file_check_sum, file_content, file_size, headers)
 
-    response = requests.post("{}".format(STORAGE_CLUSTER_ADDRESS + 'upload/'), headers = headers, data = file_content)
-    return (response.status_code, file_size, file_check_sum)
 
+def upload_file(file_content, headers):
+    response    = requests.post(
+                    "{}".format(
+                        STORAGE_CLUSTER_ADDRESS + 'upload/'
+                    ),
+                    headers = headers,
+                    data    = file_content,
+                    verify  = False,
+                  )
+    return response
 
-def get_force_get_task_result(task_id, current_time, size, checksum):
 
-    compute_task_def = message.ComputeTaskDef()
-    compute_task_def['task_id'] = task_id
-    compute_task_def['deadline'] = current_time + 60
-    task_to_compute = message.TaskToCompute(
-        compute_task_def = compute_task_def
-    )
+def get_force_get_task_result(task_id, current_time, size, checksum, task_deadline_offset=60):
+    task_to_compute      = create_task_to_compute(current_time, task_id, task_deadline_offset)
+
     report_computed_task = message.ReportComputedTask(
-        task_to_compute = task_to_compute,
-        size            = size,
-        checksum        = checksum,
+        task_to_compute  = task_to_compute,
+        size             = size,
+        checksum         = checksum,
     )
 
-    with freeze_time(timestamp_to_isoformat(current_time)):
-        force_get_task_result = message.concents.ForceGetTaskResult(
-            report_computed_task = report_computed_task,
-        )
+    force_get_task_result = message.concents.ForceGetTaskResult(
+        report_computed_task = report_computed_task,
+    )
 
     return force_get_task_result
 
 
-def main():
-    cluster_url     = parse_command_line(sys.argv)
-    current_time    = get_current_utc_timestamp()
-    task_id         = str(random.randrange(1, 100000))
-
-    (response_status_code, file_size, file_check_sum) = upload_new_file_on_cluster(
-        task_id = task_id,
-        part_id = '0',
-        current_time = current_time,
-    )
-    if response_status_code == 200:
-        print('\nCreated file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(task_id, file_check_sum, file_size))
-    else:
-        print('File has not been stored on cluster')
+class count_fails(object):
+    """
+    Decorator that wraps a test functions for intercepting assertions and counting them.
+    """
+    instances = []
+
+    def __init__(self, fun):
+        self._fun     = fun
```

Comment:
Please use whole words for variable names. I suspect this was meant to be `function`?

Reply:
done